lua/lsp/null-ls/services.lua

```diff
@@ -1,29 +1,23 @@
 local M = {}
 
-local logger = require("core.log"):get_default()
-
 local function find_root_dir()
-  if lvim.builtin.rooter.active then
-    --- use vim-rooter to set root_dir
-    vim.cmd "let root_dir = FindRootDirectory()"
-    return vim.api.nvim_get_var "root_dir"
-  end
-
-  -- TODO: Rework this to not make it javascript specific
-  --- use LSP to set root_dir
+  local util = require "lspconfig/util"
   local lsp_utils = require "lsp.utils"
+
   local ts_client = lsp_utils.get_active_client_by_ft "typescript"
```

Comment:
Does the `lsp_utils.get_active_client_by_ft` work, if there are multiple active language servers on filetype "typescript"? E.g. `tsserver`, `tailwindcss` and `null-ls`. Maybe `lsp_utils.get_active_client_by_ft` should return a list and the functionality on L8-12 should adapt to possible list?

Reply:
> Maybe lsp_utils.get_active_client_by_ft should return a list and the functionality on L8-12 should adapt to possible list?
I added a comment regarding that, so we can fix it later.

I also switched the function that I'm using to make it clearer that we're only checking if `tsserver` is active.